src/main/java/io/cdap/plugin/http/sink/batch/HTTPRecordWriter.java

```diff
@@ -178,25 +191,21 @@ private void executeHTTPService() throws IOException {
   }
 
   private String createFormMessage(StructuredRecord input) {
-    boolean first = true;
-    String formMessage = null;
-    StringBuilder sb = new StringBuilder("");
+    StringBuilder sb = new StringBuilder();
     for (Schema.Field field : input.getSchema().getFields()) {
-      if (first) {
-        first = false;
-      } else {
-        sb.append("&");
+      String key = field.getName();
+      String value = fieldToString(input, field);
```

Comment:
this is a change from before that's unrelated to json. It looks like it should be in a different commit and PR. Is there another jira open for it?

Reply:
Changes are reverted

Reply:
Changes are reverted